Lombiq.Privacy.Samples/Scripting/WorkflowHelperMethodProvider.cs

```diff
@@ -0,0 +1,42 @@
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Scripting;
+using OrchardCore.Workflows.Http.Controllers;
+using OrchardCore.Workflows.Http.Models;
+using OrchardCore.Workflows.Services;
+using System;
+using System.Collections.Generic;
+
+namespace Lombiq.Privacy.Samples.Scripting;
+
+public class WorkflowHelperMethodProvider : IGlobalMethodProvider
```

Comment:
Include some basic documentation to explain what is `workflowHttpEventUrlResolve()` and why we need it.